### ISSUE TITLE
Incorporate support for Elements List in Google Docs

### ISSUE THREAD

**Issue by bhavyashah** [nvaccess/nvda#12080]
### Is your feature request related to a problem? Please describe.
Online collaborative office suite type apps, such as the Google Workspace Apps, have become increasingly prevalent. They are used on a day-to-day basis in colleges, workplaces, and even informal situations to work on a document together. This is a broader argument to eventually port all of NVDA's functionality for Microsoft Office to Google Workspace as well. For now though, I will make my request specific to (a) Google Docs, and (b) the Elements List dialog of NVDA. Google Docs should be a priority because it is, as the word processing equivalent, the most commonly used of the Google suite. Elements List is particularly valuable, because (i) quick navigation using Google Docs' multi-step keyboard shortcuts is already inconvenient and (ii) given how incredibly common commenting is on Google Docs, viewing a list of them in an accessible dialog through NVDA would be a lifesaver.

### Describe the solution you'd like
Implement the Elemens List dialog in Google Docs. Pressing NVDA+F7 when focused inside the document (as opposed to the webpage) should open it. The list of element types should include links, headings, anotations, and errors, akin to Elements List in Microsoft Word's browse mode.

### Describe alternatives you've considered
This feature requests stems from my inability to efficiently deal with comments in Google Docs. If there is a way to jump to next and previous comment in Docs, that would help somewhat. But if there is a way to view a list of all comments in the document natively, that would make Elements List considerably less necessary for me.

### Additional context
More than 2 billion people use GSuite. A sizeable proportion of them probably just use Gmail and maybe Drive, but even if an insubstantial percentage of 2 billion is taken, that makes Google Docs a very widely used tool. Therefore, its accessibility support, I think, should be increasingly viewed as a priority for NVDA.

**Comment by Adriani90** [comment]
Elements list works now as expected in Google docs. Closing as works for me.

**Comment by lukaszgo1** [comment]
I don't think this issue is really solved. What is requested here is improvements to the elements list, so that it offers the same navigational abilities (showing all comments, spelling errors), as in Microsoft Word. Can this be reopened please?
